src/scales/ordinalScale.ts

```diff
@@ -50,76 +49,42 @@ export module Scale {
         return this;
       }
     }
 
     /**
-     * Returns the width of the range band. Only valid when rangeType is set to "bands".
+     * Returns the width of the range band.
      *
-     * @returns {number} The range band width or 0 if rangeType isn't "bands".
+     * @returns {number} The range band width
      */
     public rangeBand() : number {
       return this._d3Scale.rangeBand();
     }
 
-    public innerPadding(): number {
-      var d = this.domain();
-      if (d.length < 2) {
-        return 0;
+    public innerPadding(): number;
+    public innerPadding(innerPadding: number): Ordinal;
+    public innerPadding(innerPadding?: number): any {
+      if (innerPadding == null) {
+        var d = this.domain();
+        if (d.length < 2) {
+          return 0;
+        }
+        var step = Math.abs(this.scale(d[1]) - this.scale(d[0]));
+        return step - this.rangeBand();
       }
-      var step = Math.abs(this.scale(d[1]) - this.scale(d[0]));
-      return step - this.rangeBand();
+      this._innerPadding = innerPadding;
+      this.range(this.range());
+      this.broadcaster.broadcast();
+      return this;
     }
 
-    public fullBandStartAndWidth(v: string) {
-      var start = this.scale(v) - this.innerPadding() / 2;
-      var width = this.rangeBand() + this.innerPadding();
-      return [start, width];
-    }
-
-    /**
-     * Get the range type.
-     *
-     * @returns {string} The current range type.
-     */
-    public rangeType() : string;
-    /**
-     * Set the range type.
-     *
-     * @param {string} rangeType If provided, either "points" or "bands" indicating the
-     *     d3 method used to generate range bounds.
-     * @param {number} [outerPadding] If provided, the padding outside the range,
-     *     proportional to the range step.
-     * @param {number} [innerPadding] If provided, the padding between bands in the range,
-     *     proportional to the range step. This parameter is only used in
-     *     "bands" type ranges.
-     * @returns {Ordinal} The calling Ordinal.
-     */
-    public rangeType(rangeType: string, outerPadding?: number, innerPadding?: number) : Ordinal;
-    public rangeType(rangeType?: string, outerPadding?: number, innerPadding?: number) : any {
-      if (rangeType == null) {
-        return this._rangeType;
-      } else {
-        if(!(rangeType === "points" || rangeType === "bands")) {
-          throw new Error("Unsupported range type: " + rangeType);
-        }
-        this._rangeType = rangeType;
-        if (outerPadding != null) {
-          this._outerPadding = outerPadding;
-        }
-        if (innerPadding != null) {
-          this._innerPadding = innerPadding;
-        }
-        this.range(this.range());
-        this.broadcaster.broadcast();
-        return this;
-      }
+    public outerPadding(outerPadding: number): Ordinal {
+      this._outerPadding = outerPadding;
+      this.range(this.range());
+      this.broadcaster.broadcast();
```

Comment:
It seems that scales don't broadcast on range setting.  Not sure if I want to broadcast for just ordinalScales